### PR TITLE
Remove deprecations

### PR DESCRIPTION
**Pull request by kierenevans**
Helm and docker configurations have changed over time and we've maintained backwards compatibility.
As we prepare for v1.0.0, we should remove the backwards compatibility.

### CODE REVIEW

File: src/_base/harness/attributes/docker-base.yml [hunk 1]
```diff
@@ -55,6 +55,8 @@ attributes:
         DB_ADMIN_PASS: = @('database.root_pass')
       resources:
         memory: "2048Mi"
+        init_memory: "1024Mi"
+        migrate_memory: "1024Mi"
```
Comment (andytson-inviqa) [inline src/_base/harness/attributes/docker-base.yml:59]: I've done it another way on a project, but ok with this for now

**Review by andytson-inviqa — DISMISSED** [review]
some more work to do for a 1.0.0 milestone but can be merged into a 1.0.0 branch

File: src/spryker/_twig/docker-compose.yml/environment.yml.twig [hunk 2]
```diff
@@ -1,12 +0,0 @@
-{% for store, vhost in @('rabbitmq.vhosts') %}
```
Comment (kierenevans) [inline src/spryker/_twig/docker-compose.yml/environment.yml.twig:1]: We need to keep this file and it's includes or provide another way to have these environment variables present.

Comment (kierenevans) [reply src/spryker/_twig/docker-compose.yml/environment.yml.twig:1]: Now resolved :balloon:

**Pull request merged**